.github/workflows/lint.yml

```diff
@@ -17,6 +17,7 @@ jobs:
   lintrunner:
     uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
     with:
+      timeout-minutes: 60
```

Comment:
Huh, not sure why it's still not working. It looks like just like this snippit: https://github.com/pytorch/pytorch/blob/main/.github/workflows/slow.yml#L43

Reply:
The job is not scheduled to run

Reply:
should use 'timeout'